_projects/mohammad_project.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Improved Assessment of Brain dynamic Functional Connectivity
+status: active
+layout: project
+people:
+  - Mohammad
+---
+
+Functional Connectivity (FC) has emerged as an important measure for understanding brain function as well as a biomarker with significant potential. 
+FC assessment was initially performed under the stationarity assumption. However, growing evidence supports that the dynamical variations of FC (dynamic FC – dFC) has a critical role related to brain function (Savva et al., 2019). FC has been commonly assessed using functional Magnetic Resonance Imaging (fMRI) data. Despite its excellent spatial resolution, fMRI has a low temporal resolution, making fMRI controversial for being used for a “high-temporal-resolution-requiring” measurement like dFC. Therefore, one of the main goals of this project is to enhance the temporal resolution of fMRI data. Built upon the enhanced fMRI data, we will then be able to carry out a high-resolution dFC assessment which will promote us a deeper understanding of dFC sources and more importantly, brain functional organization.
```

Comment:
I like that you are providing references. In that case I would add either a link or DOI at the end of the document for an interested reader

Reply:
Good idea!

Reply:
I think a better word for "promote" here would be "provide"
```suggestion
FC assessment was initially performed under the stationarity assumption. However, growing evidence supports that the dynamical variations of FC (dynamic FC – dFC) has a critical role related to brain function (Savva et al., 2019). FC has been commonly assessed using functional Magnetic Resonance Imaging (fMRI) data. Despite its excellent spatial resolution, fMRI has a low temporal resolution, making fMRI controversial for being used for a “high-temporal-resolution-requiring” measurement like dFC. Therefore, one of the main goals of this project is to enhance the temporal resolution of fMRI data. Built upon the enhanced fMRI data, we will then be able to carry out a high-resolution dFC assessment which will provide with us a deeper understanding of dFC sources and more importantly, brain functional organization.
```